igvm/hypervisor_preferences.py

```diff
@@ -151,6 +151,11 @@ def __call__(self, vm, hv) -> bool:
             return False
 
         hv_model = hv.dataset_obj[self.hardware_model]
+
+        # Bail out if hardware_model is not in HYPERVISOR_CPU_THRESHOLDS list
+        if hv_model not in self.hv_cpu_thresholds:
+            return False
```

Comment:
will this pop up as a warning in igvm during hypervisor finding? if not i don't like silently ignoring a hv. it should be visible somehow that its not supported at all or? didnt @lacunoc have an opinion on this too?

Reply:
I agree that a warning would be nice here but looking at the surrounding code I see no message printed in any of the other hypervisor preferences function. 

I will test how the output looks like and report back

Reply:
> WARNING: igvm.hypervisor_preferences: Missing setting for "Dell_R640" in HYPERVISOR_CPU_THRESHOLDS
> WARNING: igvm.hypervisor_preferences: Missing setting for "Dell_R640" in HYPERVISOR_CPU_THRESHOLDS
> INFO: igvm.hypervisor_preferences: Hypervisor "xxxxxxxxxx.xxxx.ig.local" selected using 10 preferences.
> INFO: igvm.hypervisor_preferences: Hypervisor "xxxxxxxxxx.xxxx.ig.local" selected using 10 preferences.

Reply:
Fixed by adding a status message when a hypervisor is skipped due to missing hardware_model in HYPERVISOR_CPU_THRESHOLDS